doc/user story - tracking.md

```diff
@@ -0,0 +1,11 @@
+Here is the simple user story for how a movie is uploaded and tracked.
+
+1. User uploads a movie with the **upload** feature. Movie is uploaded as an MP4.
+2. User click "analyze." Screen clears and shows the first frame of the movie with the tools for adding and moving trackpoints.
+3. User identifies four points: ruler0 and ruler10 (the start and stop points on the ruler), plant1 and plant2 (the two control points on the plant)
```

Comment:
Why ruler0 and ruler10? What do those numbers mean? How about something like calibration_start calibration_end? Also need to know calibration_distance, that is, how many millimeters we think are between calibration_start and calibration_end.

Also, the idea that there are two control points is not true for all types of tracking. And for some tracking algorithms, we don't want to specify a tracking point but rather a tracking rectangle. So the control points could be points (the center of a rectangle) and then we pass in the x and y lengths for the rectangle around the point, or we specify each control rectangle with two points (for example, upper left and lower right). Admittedly this is a more general interface than we've already implemented. Perhaps the control points could be an array or list of rectangles?

Reply:
I want where the ruler is at 0mm and where it is at 10mm. And from that everything can be inferred. But we could make it "ruler 0mm" and "ruler 10mm" if that would be more clear?

The math is really easy with two control points on the ruler. It is hard with multiple control points.